spec/factory_specs/legal_aid_application_factory_spec.rb

```diff
@@ -10,16 +10,16 @@
         end
       end
 
-      context ":with_applicant specified" do
-        context ":with_bank_accounts not specified" do
+      context "with applicant specified" do
```

Comment:
I dont think these need to be changed, we are testing the traits and the traits are called` :with_applicant` or `:with_bank_accounts`

Reply:
```suggestion
      context "when :with_applicant specified" do
```
Might make the cop happy and leave the test description valid?